extension/flat_tensor/serialize/serialize.cpp

```diff
@@ -109,14 +109,38 @@ runtime::Error save_ptd(
       tensor_alignment,
       builder.CreateVector(tensors),
       builder.CreateVector(buffers));
-  builder.Finish(flat_tensor); // Our flatbuffer is created now.
+  builder.Finish(flat_tensor, ::flat_tensor_flatbuffer::FlatTensorIdentifier());
+  // Our flatbuffer is created now.
 
   // Calculate flatbuffer padding.
   auto padded_flatbufer_size =
       aligned_size(builder.GetSize(), tensor_alignment);
   auto padded_header_size =
       aligned_size(FlatTensorHeader::kHeaderExpectedLength, tensor_alignment);
 
+  // The general structure of the file is:
+  // [flatbuffer offset to root table][flat tensor magic bytes][header]
```

Comment:
"flat tensor magic bytes" is ambiguous since it could also refer to the magic in FlatTensorHeader.
"flatbuffer file identifier" would be more specific.

Similar with "header"; could say "FlatTensorHeader". And there may be some padding after the the header before the flatbuffer contents.